packages/components-native/src/Select/Select.tsx

```diff
@@ -151,15 +151,13 @@ export function Select({
           <View
             style={[styles.container, (invalid || !!error) && styles.invalid]}
           >
-            {label && (
-              <Text
-                level="textSupporting"
-                variation={textVariation}
-                hideFromScreenReader={true}
-              >
-                {label}
-              </Text>
-            )}
+            <Text
+              level="textSupporting"
+              variation={textVariation}
+              hideFromScreenReader={true}
+            >
+              {label}
+            </Text>
```

Comment:
<img width="306" alt="Screenshot 2024-02-13 at 10 52 32 PM" src="https://github.com/GetJobber/atlantis/assets/104797704/527b3b25-9a97-4d13-90d3-7e014d94ed6b">

I think we need to adjust a `flexDirection` somewhere - Selects are looking a bit wonky 🤔 

If I remove the `flexDirection: row` it looks better.  And I'm wondering if we're able to vertically center that chevron a bit more, to match... `InputPassword` for example has that eye icon vertically center

<img width="1074" alt="Screenshot 2024-02-13 at 10 54 19 PM" src="https://github.com/GetJobber/atlantis/assets/104797704/a0b08afd-7955-4c9a-aa2b-be4fa96591f9">

Reply:
The issues with Select are largely due to them having been made before InputPressable so it has weird styles from before. I think we should focus more on how it actually renders in the mobile device. Especially because the existing Select for mobile renders incorrectly as well
<img width="329" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/43e13cc6-2db7-4fdc-abdd-199def2d3d32">

Reply:
Ahh okay that's fair - Select's look good on my device so let's leave it at that!